test/Npgsql.Tests/TestUtil.cs

```diff
@@ -166,7 +166,8 @@ public static string GetUniqueIdentifier(string prefix)
         internal static Task<IAsyncDisposable> CreateTempTable(NpgsqlConnection conn, string columns, out string tableName)
         {
             tableName = "temp_table" + Interlocked.Increment(ref _tempTableCounter);
-            return conn.ExecuteNonQueryAsync($"DROP TABLE IF EXISTS {tableName} CASCADE; CREATE TABLE {tableName} ({columns})")
+            return conn.ExecuteNonQueryAsync(@$"START TRANSACTION; SELECT pg_advisory_xact_lock(0);
```

Comment:
Why is all this needed? We're guaranteed to have a unique tableName via _tempTableCounter no?

Reply:
I remember having an issue whenever multiple tables were created concurrently (something with the types). I can change it back just to test whether the issue is back...